### ISSUE TITLE
exponentially increasing loss

### ISSUE THREAD

**Issue by Sajal-1** [PacktPublishing/Deep-Learning-with-TensorFlow#7]
I am getting this output

 **FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
Train size: 3761
Validation size: 417
Test size: 1312
WARNING:tensorflow:From ./EmotionDetector_1.py:119: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See @{tf.nn.softmax_cross_entropy_with_logits_v2}.

2018-05-31 15:40:18.578886: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
WARNING:tensorflow:Passing a `GraphDef` to the SummaryWriter is deprecated. Pass a `Graph` object instead, such as `sess.graph`.
Training Loss: 13.650236
2018-05-31 15:40:20.163425 Validation Loss: 13.650264
Training Loss: 41.581100
Training Loss: 393.378937
Training Loss: 1884.418457
Training Loss: 5707.721191
Training Loss: 15281.135742**

after this it is taking too long to give any result
